aws-push-notifications-pinpoint/src/main/java/com/amplifyframework/pushnotifications/pinpoint/EventSourceType.kt

```diff
@@ -39,6 +39,8 @@ internal class EventSourceType private constructor(
         private const val AWS_EVENT_TYPE_RECEIVED_FOREGROUND = "received_foreground"
         private const val AWS_EVENT_TYPE_RECEIVED_BACKGROUND = "received_background"
 
+        private val pinpointJourneyPrefix = PushNotificationsConstants.PINPOINT_PREFIX.dropLast(1)
```

Comment:
Nit: I would suggest not tying this key to Journey, after all, it ends up just being the string `pinpoint`. If Pinpoint were to add a new feature in the future, I would expect it to follow this pattern vs the `pinpoint.whatever` one as well.

I'd also just call it key vs prefix since it's going to be treated as the whole key string
```suggestion
        private val pinpointMetadataKey = PushNotificationsConstants.PINPOINT_PREFIX.dropLast(1)
```

Reply:
I went complete 360 on this and removed the dot from prefix 😅. As you pointed out, change was needed in payload too.